src/AppInstallerRepositoryCore/Microsoft/PinningIndex.h

```diff
@@ -29,6 +29,11 @@ namespace AppInstaller::Repository::Microsoft
             return { filePath, disposition, std::move(indexFile) };
         }
 
+        // Opens or creates a PinningIndex database on the default path.

```

Comment:
nit: update comment

Reply:
Will update the comment in my other change as soon as I merge them.